redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -222,4 +222,81 @@ public static function logFile(): string
     {
         return rex_path::log('mail.log');
     }
+
+    public static function ErrorMail()
+    {
+        $addon = rex_addon::get('phpmailer');
+        rex_extension::register('RESPONSE_SHUTDOWN', static function (rex_extension_point $ep) use ($addon) {
```

Comment:
Ich würde die EP-Registrierung tendenziell in der boot.php lassen, und dann nur die innere anonyme function hierhin auslagern.
Vorteile:
- man hat in der boot.php einen überblick, auf welche EPs sich das Addon registriert
- Der Code rückt hier eine Ebene tiefer bzgl. Einrückung

Reply:
Danke übernommen.

Reply:
@gharlan aber ich glaube ich rufe den EP jetzt falsch auf

Reply:
Sieht mir richtig aus. (ggf. war dein kommentar auch vor deinen letzten änderungen?)

Reply:
danke